### PR TITLE
gh-794 - table view now truncates long values

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

File: ui/src/main/webapp/app/table/table-component.js [hunk 1]
```diff
@@ -308,6 +309,24 @@ function TableController(schema, results, table, events, common, types, time, cs
         return '"' + vm.sortType + '"';
     }
 
+    vm.shouldShowTruncation = function(value) {
+        return value && typeof value === 'string' && value.length > maxValueLength;
+    }
+
+    vm.getTruncatedValue = function(value) {
+        if(vm.shouldShowTruncation(value)) {
+            return value.substring(0,maxValueLength-3);
```
Comment (d47853) [inline ui/src/main/webapp/app/table/table-component.js]: You may want to add an ellipsis to show the value is truncated

Comment (p013570) [reply ui/src/main/webapp/app/table/table-component.js]: The ellipsis are added in the table.html file.

File: ui/src/main/webapp/app/table/table-component.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ function resultsTable() {
  */
 function TableController(schema, results, table, events, common, types, time, csv, $mdDialog) {
     var vm = this;
+    var maxValueLength = 500;
```
[the remark this answers is not in the document]
Comment (p013570) [reply ui/src/main/webapp/app/table/table-component.js]: Yeah that would be useful.

File: ui/src/main/webapp/app/table/table-component.js (hunk 1 above)
Comment (d47853) [reply ui/src/main/webapp/app/table/table-component.js]: Oh yeah. Sorry I missed that

File: ui/src/main/webapp/app/table/table-component.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (p013570) [reply ui/src/main/webapp/app/table/table-component.js]: @d47853  - I've made the requested change.

**Pull request merged**